lib/createStylelint.cjs

```diff
@@ -11,7 +11,7 @@ const IS_TEST = process.env.NODE_ENV === 'test';
 const STOP_DIR = IS_TEST ? process.cwd() : undefined;
 
 /**
- * @type {import('stylelint')['_createLinter']}
+ * @type {import('stylelint').PublicApi['_createLinter']}
```

Comment:
nit

It's kind of weird to access something that is expected to be private through `PublicApi`.